CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## Unreleased
+
+- **Breaking change**: The `Envelope` type can now be an empty envelope.
+  Previously, it was only able to represent a rectangle with some area, a
+  horizontal or vertical line, or a single point. Its `AsGeometry` returns
+  an empty `GeometryCollection` in the case where it's empty. The result of
+  `AsGeometry` is unchanged for non-empty envelopes.
+
+- **Breaking change**: The `NewEnvelope` function signature has changed. It now
+  accepts a slice of `geom.XY` as the sole argument. The behaviour of the
+  function is the same as before, except that if no XY values are provided then
+  an empty envelope is returned.
```

Comment:
```suggestion
  an empty envelope is returned without error.
```

Reply:
Good suggestion, fixed in 1e47e80742c5cce93476196eaa3a0be051c1c3de